reference/price_lists.v3.yml

```diff
@@ -2935,19 +3022,18 @@ components:
       x-internal: false
     PriceListAssignmentsIdentifiers:
       type: object
-      description: '`Price List Assignments` object used in GET response.'
       allOf:
         - type: object
           properties:
             customer_group_id:
               type: integer
-              description: The Customer Group with which this price list assignment is associated. A customer_group_id of 0 indicates default pricing for the channel given in the request.
+              description: The customer group with which this price list assignment is associated. A `customer_group_id` of 0 indicates default pricing for the channel given in the request.
             price_list_id:
               type: integer
-              description: The Price List with which this price set is associated.
+              description: The price list with which this price set is associated.
             channel_id:
               type: integer
-              description: The Channel with which this price set is associated. This field is optional or may be null when a price list applies across channels for the customer_group_id in the assignment.
+              description: The channel with which this price set is associated. This field is optional or may be `null` when a price list applies across channels for the `customer_group_id` in the assignment.
```

Comment:
Should these be capitalized? I thought it looked over capitalized, but I also know that UI names should be in caps

Reply:
Just found it in the UI, so I'll capitalize